app/views/projects/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<h2><%= @project.name %></h2>
+<h4>Material: <%= @project.material %></h4>
+<h4>Challenge Theme: <%= @project.project_challenge_theme %></h4>
+<h5>Number of Contestants: <%= @project.contestant_count %></h5>
+<h5>Average Contestant Experience: <%= @project.average_contestant_experience %> years</h5>
+<%= form_with url: "/contestant_projects/#{@project.id}", method: :post, local: true, data: {turbo: false} do |f| %>
```

Comment:
Nice RESTful path! When we have an ID with this type of path that lists the joins record (contestant_projects) I would assume the ID following it would be the ID of the joins record. In this case, though, it's the ID of the project (which is more readily available). So, a path that might make more sense here (and is equally RESTful) is `"/projects/:project_id/contestants"`